### PR TITLE
Update Readme instructions

### PR DESCRIPTION
**Pull request by oindrillac**
Updated instructions for contributing based on changes in #117

### CODE REVIEW

**Review by tumido — APPROVED** [review]
LGTM, nicely written and easy to follow! :raised_hands:

File: README.md [hunk 1]
```diff
@@ -1,63 +1,87 @@
...
+    <JupyterNotebook path="notebook.ipynb"/>
+    ```
+
+## Location of Content
 
-#### Adding remote content
+Content will be added to one of the four categories on this website:
+
+* **data-science**: Examples of data science projects for the data science users that want to learn about data science on ODH.
+* **users**: Documentation for all users of ODH. Access details of various deploed ODH components.
```
Comment (hemajv) [inline README.md]: change `deploed` to `deployed`

File: README.md [hunk 2]
```diff
@@ -1,63 +1,87 @@
...
 ```shell script
 git clone git@github.com:operate-first/continuous-deployment.git
 ```
 Include the remote repository to `.gitignore`.
 
-Create a separate folder within the remote repository which only consists of the document(s) to be published and its supporting files like images.
+Add remote repository url and directory to get content from in `content-sources.yaml`. 
 
-**If adding remote notebooks**, it's required to create an *.mdx file in your remote repository's publishing folder for each notebook you want to render (this file is what actually gets displayed). We recommend the following naming convention `notebook-you-want-to-publish-name.mdx` .
+Add the Git source where you are adding content from. Set the name of the repo, the root ad the `dir` and set the urlPrefix as the content category you are adding that content to (see the four categories mentioned above) and the name of the repository.
```
Comment (hemajv) [inline README.md]: is `the root ad` supposed to be `the root address`?

File: README.md [hunk 3]
```diff
@@ -80,35 +104,40 @@ The *.mdx files should follow the format outlined in the example above. However,
...
-  gitSrc: https://github.com/operate-first/continuous-deployment.git
-  dir: continuous-deployment/docs/publish
-  urlPrefix: cd
-```
+For example, to add content locally to the `blueprints` category, create an index.md located at `content/blueprints/content-name/index.md`.
+
+### Configuring Table of Contents
 
-There are two ways you can manage the table of contents for the remote repo.
+Whether adding content remotely, or locally, the content needs to be added to the vertical navigation bar on the left and belongs to one of the four categories.
```
Comment (hemajv) [inline README.md]: `and belongs` to `should belong`?

**Review by hemajv — CHANGES_REQUESTED** [review]
apart from the minor fixes, lgtm! :ship:

File: README.md (hunk 1 above)
Comment (oindrillac) [reply README.md]: changed

File: README.md (hunk 3 above)
Comment (oindrillac) [reply README.md]: :+1:

File: README.md (hunk 2 above)
Comment (oindrillac) [reply README.md]: changed

**Comment by oindrillac** [comment]
@hemajv made the changes

File: README.md [hunk 4]
```diff
@@ -1,63 +1,87 @@
...
 ```shell script
 git clone git@github.com:operate-first/continuous-deployment.git
 ```
 Include the remote repository to `.gitignore`.
 
-Create a separate folder within the remote repository which only consists of the document(s) to be published and its supporting files like images.
+Add remote repository url and directory to get content from in `content-sources.yaml`. 
 
-**If adding remote notebooks**, it's required to create an *.mdx file in your remote repository's publishing folder for each notebook you want to render (this file is what actually gets displayed). We recommend the following naming convention `notebook-you-want-to-publish-name.mdx` .
+Add the Git source where you are adding content from. Set the name of the repo, (ie. the root directory of the repo) the `dir` and set the urlPrefix as the content category you are adding that content to (see the four categories mentioned above) and the name of the repository.
```
Comment (hemajv) [inline README.md]: should `(ie. the root directory of the repo)` come after `the dir`?

Comment (oindrillac) [reply README.md]: ah yes, changed

**Review by hemajv — APPROVED** [review]
:ship:

**Pull request merged**